static/styles/subscriptions.css

```diff
@@ -497,13 +497,13 @@ h4.stream_setting_subsection_title {
         background-repeat: no-repeat;
         background-position: center center;
 
-        svg {
+        .tick {
```

Comment:
I think it might be better to use `zulip-icon-check` here, rather than introducing a new CSS class.

Reply:
Updated the PR to remove `.tick`.